### PR TITLE
Completed: Alert

### PR DESCRIPTION
**Pull request by hemanshthakur**

### CODE REVIEW

**Comment by hemanshthakur** [comment]
alert-showcase

File: containers/css/alert.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    background-color: var(--neutral-color);
+    color:var(--primary-color);
+    font-weight: bolder;
+}
+.text-in-alert{
+    margin-left: 1rem;
+}
+.btn-in-alert{
+    margin-right: 1rem;
+    background: var(--primary-color);
```
Comment (Bibin-Antony) [inline containers/css/alert.css]: put these into a common class container

**Comment by Bibin-Antony** [comment]
could try and center the close button center, everything else seems fine could proceed.

File: containers/html/alert.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+</head>
+<body>
+    <div class="alert-containers">
+        <div class="alert">
+            <p class="text-in-alert">Here is some text!</p>
+            <button class="btn-in-alert">x</button>
+        </div>
+        <div class="alert success-div">
+            <p class="text-in-alert">Here is some text!</p>
+            <button class="btn-in-alert success-btn">x</button>
```
Comment (hiteshverma27) [inline containers/html/alert.html]: Try to use a standard naming convention like BEM so that it is easy to understand.

**Review by hiteshverma27 — APPROVED** [review]

**Review by Bibin-Antony — APPROVED** [review]

**Review by Bibin-Antony — APPROVED** [review]

**Pull request merged**